### ISSUE TITLE
Plugin makes syntax invalid

### ISSUE THREAD

**Issue by demaisj** [prettier/plugin-pug#27]
Hello, today while setting up prettier + prettier-pug I encountered this problem:

The prettier-pug plugin, when ran on a pug file with the --write flag, will fix the code style and as a side-effect sometimes corrupt the syntax of the file, producing an invalid file which cannot be processed on subsequent runs.

As an example here this completely valid pug file:
```pug
tag(key='value') #{identifier}
tag
  | #{identifier}
```

The prettier-pug produces this output, which is invalid:
```pug
tag(key='value')#{identifier}
tag
#{identifier}
```

When using the pug package, it reports this error:
```
Error: /file.pug:1:17
  > 1| tag(key='value')#{identifier}
-----------------------^
    2| tag
    3| #{identifier}
    4| 
  
Unexpected token `interpolation` expected `text`, `interpolated-code`, `code`, `:`, `slash`, `newline` or `eos`
```

When running pug-prettier on the new pug file, it produces this error:
```
[error] /file.pug: Error: Unhandled token: {"type":"interpolation","loc":{"start":{"line":1,"column":17},"end":{"line":1,"column":30}},"val":"identifier"}
[error]     at _loop_1 (/node_modules/@prettier/plugin-pug/dist/index.js:542:35)
[error]     at Object.print (/node_modules/@prettier/plugin-pug/dist/index.js:546:21)
[error]     at callPluginPrintFunction (/node_modules/prettier/bin-prettier.js:13622:18)
[error]     at /node_modules/prettier/bin-prettier.js:13573:16
[error]     at Object.printComments (/node_modules/prettier/bin-prettier.js:13295:17)
[error]     at printGenerically (/node_modules/prettier/bin-prettier.js:13572:22)
[error]     at printAstToDoc (/node_modules/prettier/bin-prettier.js:13584:16)
[error]     at coreFormat (/node_modules/prettier/bin-prettier.js:13903:16)
[error]     at format (/node_modules/prettier/bin-prettier.js:14148:73)
[error]     at formatWithCursor (/node_modules/prettier/bin-prettier.js:14164:12)
```

**Comment by Shinigami92** [comment]
I will work on it in two days

**Comment by Shinigami92** [comment]
Released as [1.0.5](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.0.5)
Thanks for using my Plugin 😃

If something is not working correctly, please reply to this issue.
